disputer/disputer.js

```diff
@@ -76,10 +76,12 @@ class Disputer {
     const undisputedLiquidations = this.empClient.getUndisputedLiquidations();
     const disputeableLiquidations = undisputedLiquidations.filter(
       liquidation =>
+        this.priceFeed.getHistoricalPrice(parseInt(liquidation.liquidationTime.toString())) &&
```

Comment:
this just became quite hard to read I think. having three nested &&'s is a bit complex. could you add a comment here perhaps?